packages/playground/src/explorer/utils/types.ts

```diff
@@ -121,3 +126,23 @@ export const nodeInitializer: GridNode = {
   cards: [],
   num_gpu: 0,
 };
+
+export type FarmFilterInputs = {
+  farmId: NodeInputFilterType;
+  name: NodeInputFilterType;
+  twinId: NodeInputFilterType;
+  certificationType: NodeInputFilterType;
+  pricingPolicyId: NodeInputFilterType;
+};
+
+export type FarmFilterOptions = {
+  page: number;
+  size: number;
+  retCount: boolean;
+};
```

Comment:
we don't need to create a type for the page and size, they can be `refs`, I've created them in the `explorer/nodes` because I have another field so I chose to include them in one interface.